app/jobs/scheduled/problem_checks.rb

```diff
@@ -5,6 +5,8 @@ module Jobs
   # on a regular basis. To add a problem check for this scheduled job run
   # call AdminDashboardData.add_scheduled_problem_check
   class ProblemChecks < ::Jobs::Scheduled
+    sidekiq_options retry: false
```

Comment:
Figured we not retry this since it's scheduled for every 10 minutes anyway.

Reply:
Yep makes sense 👍

Reply:
It's pretty unlikely now to fail anyway since it's just fanning jobs out